graql/reasoner/value-predicate.feature

```diff
@@ -769,12 +769,12 @@ Feature: Value Predicate Resolution
       """
       insert
 
-      $x isa post, has creation-date 2020-07-01;
-      $x1 isa post, has creation-date 2020-07-02;
-      $x2 isa post, has creation-date 2020-07-03;
-      $x3 isa post, has creation-date 2020-07-04;
-      $x4 isa post, has creation-date 2020-07-05;
-      $x5 isa post, has creation-date 2020-07-06;
+      $x isa post, has creation-date 2021-07-01;
```

Comment:
this intended @vmax ? probably is fnie haha

Reply:
good to review your own PR before considering it done :)

Reply:
yes let's revert it

Reply:
@flyingsilverfin @lolski addressed in https://github.com/graknlabs/behaviour/pull/136/commits/98f340f72238de28ce14dc2f629c82b1b1fc6242